tool/dev.dart

```diff
@@ -22,7 +22,7 @@ main(args) async {
   config.copyLicense.directories = directories;
   config.coverage.reportOn = ['bin/', 'lib/'];
   config.format
-    ..directories = directories
+    ..paths = directories
```

Comment:
Addressed in https://github.com/greglittlefield-wf/dart_dev/pull/1/commits/6f8dde6e7c7ebed5f0e0fc5c38cca4455f962938